lib/matplotlib/axes/_axes.py

```diff
@@ -5291,9 +5291,7 @@ def get_interp_point(idx):
                             np.column_stack([ind[where], dep2[where]])])
         if ind_dir == "y":
             pts = pts[:, ::-1]
-        self.dataLim.update_from_data_xy(pts, self.ignore_existing_data_limits,
```

Comment:
Should this method get a comment that its usually meant to be called by `Axes.update_datalim`?  We need more breadcrumbs for developers so they don't get lost amongst the many levels of redirection....